data/mods/Aftershock/items/ammo/ballistic.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "id": "afs_25mm_caseless",
+    "type": "AMMO",
+    "name": { "str_sp": "25x40mm shot, caseless" },
+    "description": "25x40mm caseless rounds.  The most common type of ammo used for high gauge shotguns and light grenade launchers.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "weight": "130 g",
+    "volume": "230 ml",
+    "price": 225,
+    "price_postapoc": 8000,
+    "flags": [ "IRREPLACEABLE_CONSUMABLE" ],
+    "material": [ "plastic", "powder" ],
+    "symbol": "=",
+    "color": "pink",
+    "count": 6,
+    "stack_size": 1,
+    "ammo_type": "afs_25mm",
+    "range": 24,
+    "damage": { "damage_type": "bullet", "amount": 65, "armor_penetration": 18 },
+    "dispersion": 75,
+    "recoil": 2200,
+    "effects": [ "NEVER_MISFIRES" ]
+  },
+  {
+    "id": "afs_10mm_caseless",
+    "type": "AMMO",
+    "name": { "str_sp": "10mm FMJ caseless" },
+    "description": "wip do this lol",
+    "weight": "9 g",
+    "volume": "117 ml",
+    "price": 400,
+    "price_postapoc": 800,
+    "flags": [ "IRREPLACEABLE_CONSUMABLE" ],
+    "material": [ "brass", "powder" ],
+    "symbol": "=",
+    "color": "yellow",
+    "count": 40,
+    "stack_size": 40,
+    "ammo_type": "afs_10mm",
+    "range": 14,
+    "damage": { "damage_type": "bullet", "amount": 31, "armor_penetration": 4 },
+    "dispersion": 50,
+    "recoil": 750,
+    "effects": [ "COOKOFF" ]
+  },
+  {
+    "id": "afs_6.5mm_caseless",
+    "type": "AMMO",
+    "name": { "str_sp": "6.5mm caseless" },
+    "description": "wip do this",
+    "weight": "12 g",
+    "volume": "194 ml",
+    "price": 280,
+    "price_postapoc": 900,
+    "flags": [ "IRREPLACEABLE_CONSUMABLE" ],
```

Comment:
This flag means the item becomes more expensive as the game calendar progresses, to imply the DDA post-apocalypse cant produce more. Not fully sure you intended to add it?

Reply:
Ahhh didn't know that part; Let me remove it.